src/modules/QtQml/Connections.js

```diff
@@ -0,0 +1,41 @@
+/* Connections */
+registerQmlType({
+  module: 'QtQml',
+  name: 'Connections',
+  versions: /.*/,
+  baseClass: 'QtObject',
+  constructor: function QMLConnections(meta) {
+    callSuper(this, meta);
+
+    createProperty("QtObject", this, "target");
+    createProperty("bool", this, "ignoreUnknownSignals");
```

Comment:
Could you port this to the new module definition syntax? See [modules/QtQuick/Animation.js](https://github.com/qmlweb/qmlweb/blob/master/src/modules/QtQuick/Animation.js).